core/helpers/EEH_Activation.helper.php

```diff
@@ -1349,7 +1349,7 @@ protected static function _activate_and_generate_default_messengers_and_message_
             // passing off to create templates (after the refactor is done).
             // @todo remove this comment when determined not necessary.
             $message_resource_manager->activate_messenger(
-                $messenger_to_generate->name,
+                $messenger_to_generate,
```

Comment:
lolz... ya let's use the object to get the name so that `activate_messenger()` can use the name to get the object!!! 

# 😆 🤔 😞